spec/benchmark-spec.js

```diff
@@ -0,0 +1,21 @@
+/** @babel */
+/* eslint-env jasmine, atomtest */
+
+/* This file contains all specs to test the startup/load-performance indicated by THE timecop. */
+
+describe('editorconfig', () => {
+	beforeEach(() => {
+		waitsForPromise(() => atom.packages.activatePackage('editorconfig'));
+	});
+
+	describe('when activated', () => {
+		const loadLimit = 50; // sets the limit for package-loading in ms.
+		it(`should being loaded in less than ${loadLimit}ms`, () => {
+			expect(atom.packages.getLoadedPackage('editorconfig').loadTime).toBeLessThan(loadLimit);
```

Comment:
It might be better to get a report of how long things took, but not necessarily Pass/Fail on it as it is so subjective. Then if you see a spike in performance, you can verify it locally before you throw the baby out with the bathwater.